ui/imports/utils/Constants.qml

```diff
@@ -396,6 +396,14 @@ QtObject {
         readonly property int communityChat: 6
     }
 
+    readonly property QtObject memberRole: QtObject{
+        readonly property int none: 0
+        readonly property int owner: 1
+        readonly property int manageUsers: 2
+        readonly property int moderateContent: 3
+        readonly property int admin: 4
```

Comment:
I just thought, I think later we might be supporting having multiple roles per user, at least if we plan on doing like Discord. Just an FYI

Reply:
Also, shouldn't this be a bitfield?